src/greenpak/i2c.py

```diff
@@ -48,11 +45,40 @@ def read(
         """
         assert False, f"Class {self.__class__} does not implement read()"
 
+    def write_read(self, addr: int, wreq: bytearray, rbytes: int,
```

Comment:
Can you have two new methods for read and write that will replace down the road the old ones.  The new method have an explicit start offset rather than hiding it in the write data.  Once you add them, I will implement them for the I2cAdapter and I2cDriver and will delete the old ones.

def gp_write(self, addr: int, start: int, data: bytearray, silent: bool = False) -> bool

read(
        self, addr: int, start: int, byte_count: int, silent: bool = False
    ) -> bytearray | None:

Reply:
Ok, will be done (but might be over weekend).
> Note that the slient is gone since there is a specific scan method to check if the device response.
Could you point to me the 'slient is gone' to me in code? I so click-on faster to what you referring to.